src/actions/breakpoints/addBreakpoint.js

```diff
@@ -90,9 +96,15 @@ async function addBreakpointPromise(getState, client, sourceMaps, breakpoint) {
     text,
     originalText
   };
-
+  if (!shiftKey) {
```

Comment:
I think `assertBreakpoint(newBreakpoint)` could be moved to be immediately after `const newBreakpoint = {...}` since it doesn't check `newBreakpoint.disabled` at all. The way it is right now makes me uncertain whether the `assertBreakpoint(newBreakpoint)` relies on `newBreakpoint.disabled`.